### ISSUE TITLE
instagram

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/social#4]

**Comment by atherdon** [comment]
#1

**Comment by mahimamishra** [comment]
Have you yet created a Instagram page?

**Comment by atherdon** [comment]
nope. i tried my 2 emails and it told me that this emails are not empty.
so i decide to wait sometime - maybe instagram is buggy right now


On 06-Jun-2018 at 02:25 AM, Mahima Mishra wrote:


Have you yet created a Instagram page?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/GroceriStar/social/issues/4#issuecomment-394919470>,
or mute
the thread

**Comment by mahimamishra** [comment]
Okay not a problem
